plugin/gcp/computeDir.go

```diff
@@ -0,0 +1,68 @@
+package gcp
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/puppetlabs/wash/plugin"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/option"
+)
+
+type computeProjectService struct {
+	*compute.Service
+	projectID string
+}
+
+type computeDir struct {
+	plugin.EntryBase
+	computeProjectService
```

Comment:
Nit: Think it's better if embedding's only restricted to `EntryBase`. That way, it's easier to see that this is strictly an Entry, and that we're just passing the project service object around. The pattern's also consistent with the other core-plugins.